container/container.go

```diff
@@ -58,136 +83,169 @@ func New(opts ...Option) *Container {
 	return cnr
 }
 
-// ToV2 returns the v2 Container message.
-//
-// Nil Container converts to nil.
-func (c *Container) ToV2() *container.Container {
+// CalculateID calculates container identifier
+// based on its structure.
+func CalculateID(c *Container) *cid.ID {
```

Comment:
We must make it a method since versioning is encapsulated in `Container` type.

Reply:
did not get your point, what does "versioning" mean? all methods called in the body of the `CalculateID` are exported